bedrock/firefox/helpers.py

```diff
@@ -145,14 +152,15 @@ def download_firefox(ctx, channel='release', small=False, icon=True,
     data = {
         'locale_name': locale_name,
         'version': version,
-        'product': 'firefox-android' if platform == 'android' else 'firefox',
+        'product': 'firefox-%s' % platform,
```

Comment:
As far as I can tell, this entire `data` hash is sent to the `download-button.html` template and that's it. I don't see any spot in that template where the `product` value is used, so...maybe this change has zero impact?